vdp/pipeline/v1beta/pipeline.proto

```diff
@@ -1984,6 +1984,26 @@ message ListPipelineRunsRequest {
   optional string order_by = 7 [(google.api.field_behavior) = OPTIONAL];
 }
 
+// ListPipelineRunsByCreditOwnerRequest is the request message for ListPipelineRunsByCreditOwner.
+message ListPipelineRunsByCreditOwnerRequest {
```

Comment:
`CreditOwner` is a cloud-only concept. Why don't we use `Requester`?

Reply:
I could rename it to `ListPipelineRunsByRequesterRequest`.